### DISCUSSION TITLE
How to remove / not display breadcrumb correct way

### DISCUSSION THREAD

**Discussion by Jimmi08** [e107inc/e107#5216] [Q&A Core]
Hi,
I have a page with error message.  From theme there are breadcrumb by default. 

It looks bad with just home icon, so I wanted to hide this element.

Only way I found is:

```
if(!USER) {
	require_once(HEADERF);
	e107::breadcrumb("anystring");
	e107::getRender()->tablerender("", "<div style='text-align:center'>" . LAN_USER_55 . "</div>");
	require_once(FOOTERF);
	exit;
}
```
it works thanks to this in breadcrumb():
`	if(!is_array($array)){ return; }`

I would suppose something like this:
```
e107::breadcrumb("");
e107::breadcrumb(NULL); 
```

for "resetting" breadcrumbs, but it doesn't work.

Or what is the recommended way?
Thanks

**Comment by tgtje** [comment]
PM  php  line 690 ?? maybe as coding or by css??  ([example read](https://thethemefoundry.com/tutorials/hiding-parts-of-your-theme-with-displaynone/#:~:text=One%20of%20the%20more%20powerful,using%20CSS%20to%20hide%20it.))

**Reply by Jimmi08** [reply]
Hi, it is not css issue.  It shouldn't be rendered because it depends on plugin, not on theme.
